cmd/orb.go

```diff
@@ -657,8 +657,7 @@ func orbInfo(opts orbOptions) error {
 		fmt.Printf("Latest: %s@%s\n", info.Orb.Name, info.Orb.HighestVersion)
 		fmt.Printf("Last-updated: %s\n", info.Orb.Versions[0].CreatedAt)
 		fmt.Printf("Created: %s\n", info.Orb.CreatedAt)
-		firstRelease := info.Orb.Versions[len(info.Orb.Versions)-1]
-		fmt.Printf("First-release: %s @ %s\n", firstRelease.Version, firstRelease.CreatedAt)
+		// firstRelease := info.Orb.Versions[len(info.Orb.Versions)-1]
 
 		fmt.Printf("Total-revisions: %d\n", len(info.Orb.Versions))
```

Comment:
If I understand the bug correctly, then this line of output is also incorrect, right? The `len(info.Orb.Versions)` shows the number that were downloaded, not the numer in total?